### PR TITLE
adaptations for PR 123 on rtt

### PR DESCRIPTION
**Pull request by planthaber**
Don't merge yet!

If PR https://github.com/orocos-toolchain/rtt/pull/123 is merged, we need these changes to make the ruby binding working again

### CODE REVIEW

**Comment by planthaber** [comment]
Updated based meyerj's comment here:
https://github.com/orocos-toolchain/rtt/pull/123

Compatible to older toolchain versions < 2.9.

I don't know if it is actually needed to use the RTT_VERSION_GTE(2,9,0) marco , as checkouts on the master/stable checkouts of rock should be consistent nevertheless. 

Waiting for the orocos-toolchain/rtt#123 PR is not required anymore.

**Comment by planthaber** [comment]
@doudou Are there other "rock bindings" I forgot or is this all which are affected by the PR linked above?

File: ext/rorocos/rorocos.cc [hunk 1]
```diff
@@ -204,8 +204,8 @@ static VALUE task_context_property_names(VALUE self)
         corba_blocking_fct_call_with_result(bind(&_objref_CConfigurationInterface::getPropertyList,(_objref_CConfigurationInterface*)context.main_service));
     for (unsigned int i = 0; i != names->length(); ++i)
     {
-        CORBA::String_var name = names[i].name;
-        rb_ary_push(result, rb_str_new2(name));
+       	CORBA::String_var name = names[i].name;
```
Comment (doudou) [inline ext/rorocos/rorocos.cc]: Bogus indentation

File: ext/rorocos/rorocos.cc [hunk 2]
```diff
@@ -219,7 +219,11 @@ static VALUE task_context_attribute_names(VALUE self)
         corba_blocking_fct_call_with_result(bind(&_objref_CConfigurationInterface::getAttributeList,(_objref_CConfigurationInterface*)context.main_service));
     for (unsigned int i = 0; i != names->length(); ++i)
     {
-        CORBA::String_var name = names[i];
+	#if RTT_VERSION_GTE(2,9,0)
+		CORBA::String_var name = names[i].name;
```
Comment (doudou) [inline ext/rorocos/rorocos.cc]: Bogus indentation. You use 4-spaces tabs and 8 is the standard. Indent with spaces.

**Comment by planthaber** [comment]
Fixed identation by force-push

**Comment by doudou** [comment]
Merged, thanks.

**Pull request merged**